test/mock/MockUSDC.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.13;
+
+import {MockERC20} from "./MockERC20.sol";
```

Comment:
```suggestion
import {MockERC20} from "solmate/test/utils/mocks/MockERC20.sol";
```

Reply:
addressed in [this](https://github.com/lindy-labs/sandclock-contracts/pull/56/commits/2a6e386c830954963fb98ddd98e1ab110a8b2b35) commit